Code/Editor/Controls/ConsoleSCB.cpp

```diff
@@ -1096,6 +1128,40 @@ AZ::ConsoleCommandInvokedEvent::Handler ConsoleVariableEditor::m_commandInvokedH
         auto changedCVar = GetIEditor()->GetSystem()->GetIConsole()->GetCVar(AZStd::string(command).c_str());
         if (changedCVar)
         {
+            auto console = AZ::Interface<AZ::IConsole>::Get();
+            auto azConsoleCommand = console->FindCommand(command);
+            if (azConsoleCommand)
+            {
+                // ints AZ::s8 AZ::u16, AZ::s16, 32, 64, long, unsigned long
+                // floats float, double,
+                // AZStd::string, AZ::CVarFixedString
+
+                const bool handled =
+                    (SetCVarFromConsoleCommand<AZ::s8, CVAR_INT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<AZ::s16, CVAR_INT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<AZ::s32, CVAR_INT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<AZ::s64, CVAR_INT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<long, CVAR_INT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<unsigned long, CVAR_INT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<float, CVAR_FLOAT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<double, CVAR_FLOAT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<AZStd::string, CVAR_STRING>(changedCVar, azConsoleCommand));
```

Comment:
You should add a case for AZ::CVarFixedString as well.

Reply:
done, thanks